src/xdocs/google_style.xml

```diff
@@ -1513,6 +1512,25 @@
                                     href="https://github.com/checkstyle/checkstyle/blob/master/src/it/java/com/google/checkstyle/test/chapter7javadoc/rule731selfexplanatory/JavadocMethodTest.java">test</a>
                             </td>
                         </tr>
+                        <tr>
+                            <td><a name="7.3.4"/><a href="#7.3.4"><img src="images/anchor.png" alt=""/></a></td>
+                            <td>
+                                <a
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s7.3.4-javadoc-non-required">7.3.4 Non-required Javadoc</a>
```

Comment:
@romani How is this report generated?
Current link isn't valid. Is that something that has to be updated in this PR?